src/python/stup/core/core.py

```diff
@@ -57,8 +60,8 @@ def __init__(self, protocol):
         self.last_recv_ack_id = -1
         self.last_recv_ack_cnt = 0
 
-        self.time_wheel_fine   = TwistedUtils.TimeWheel(255, Config.TIMER_FINE_GRANULARITY)
-        self.time_wheel_coarse = TwistedUtils.TimeWheel(10, Config.TIMER_COARSE_GRANULARITY)
+        self.time_wheel_fine   = twisted.TimeWheel(255, Config.TIMER_FINE_GRANULARITY)
```

Comment:
I don't think `stup.twisted` is a good name because it might lead to a confusion that this module is from original twisted code. `twistedutils` is something better here.